portshaker.subr.in

```diff
@@ -10,8 +10,10 @@ if [ -z "${_portshaker_subr_loaded}" ]; then
 _portshaker_subr_loaded="YES"
 verbose="0"
 
+PORTSHAKER_CONFIG_DIR="${PORTSHAKER_CONFIG_DIR:=@@ETCDIR@@}"
+
 if [ "${_portshaker_load_config}" != "no" ]; then
-	config_dir="@@ETCDIR@@"
+	config_dir="${PORTSHAKER_CONFIG_DIR}"
```

Comment:
With such a change, the new variable `$PORTSHAKER_CONFIG_DIR` is a duplicate of `$config_dir`.

For some reason I can't remember `portshaker(8)` is sensible to lower cased environment variables, documented in [portshaker.8.in](https://github.com/smortex/portshaker/blob/master/portshaker.8.in) (`$portshaker_info`, `$portshaker_debug`).  Renaming `$PORTSHAKER_CONFIG_DIR` to lowercase and describing it in the man page would make things consistent.

The current code of the PR could then be simplified like this:

~~~sh
config_dir="${portshaker_config_dir:=@@ETCDIR@@}"
~~~